origin-contracts/test-alt/TokenMigration.js

```diff
@@ -35,10 +35,39 @@ describe('TokenMigration.sol', async function() {
       path: `${contractPath}/token/`,
       args: [OldToken._address, NewToken._address]
     })
+    await OldToken.methods.pause().send({from: owner})
+    await NewToken.methods.pause().send({from: owner})
+
     await NewToken.methods.transferOwnership(TokenMigration._address).send({from: owner})
     assert.equal(await TokenMigration.methods.finished().call(), false)
   })
 
+
+  it('disallow migrating when fromToken is not paused', async function() {
```

Comment:
Thanks for adding this test. Could you also add an equivalent for ensuring that `toToken` must be paused?